test/tests/functional/pbs_ralter.py

```diff
@@ -325,21 +322,21 @@ def alter_a_reservation(self, rid, start, end, shift,
 
             if interactive:
                 if whichMessage == 1:
-                    msg = "pbs_ralter: " + rid + " CONFIRMED"
+                    msg = "pbs_ralter: " + r + " CONFIRMED"
                 elif whichMessage == 2:
-                    msg = "pbs_ralter: " + rid + " UNCONFIRMED"
+                    msg = "pbs_ralter: " + r + " UNCONFIRMED"
                 else:
-                    msg = "pbs_ralter: " + rid + " DENIED"
+                    msg = "pbs_ralter: " + r + " DENIED"
             else:
-                msg = "pbs_ralter: " + rid + " ALTER REQUESTED"
+                msg = "pbs_ralter: " + r + " ALTER REQUESTED"
 
-            ret = self.server.alterresv(rid, attrs)
-            self.assertEqual(ret['rc'], 0)
+            rc = self.server.alterresv(r, attrs)
+            self.assertEqual(rc, 0)
```

Comment:
Do you need check for this?  Won't alterresv() raise an exception if something bad happened?

Reply:
Removed this check.